src/main/java/org/jenkinsci/plugins/fodupload/polling/ScanStatusPoller.java

```diff
@@ -257,6 +257,11 @@ private void processScanRelease() {
         {
             fail = true;
         } else {
+            if(statusString.equals(AnalysisStatusTypeEnum.InProgress.name())) {
+                result.setScanInProgress(true);
+                result.setPassing(releaseDTO.isPassed());
+                finished = true;
```

Comment:
Are we stopping polling here when we find the scan is still in progress?

Reply:
The Jenkins pipeline should not wait on in progress scans because that can take weeks. Even if it's in queue, it needs to end polling and allow the dev to finish his pipeline/freestyle job.